src/server/services/landingpage/qgslandingpageutils.h

```diff
@@ -78,6 +78,16 @@ struct QgsLandingPageUtils
    */
   static QString projectUriFromUrl( const QString &url );
 
+  /**
+   * Store env configuration parameter, used to invalidate the cache if the value changes
+   */
+  static QString QGIS_SERVER_PROJECTS_DIRECTORIES;
```

Comment:
Can these be made `Q_GLOBAL_STATIC` instead? https://github.com/qgis/QGIS/pull/32552

Reply:
I think I'll make them local static instead...